packages/appstore/src/constants/wallet-steps-config.tsx

```diff
@@ -0,0 +1,42 @@
+import { localize } from '@deriv/translations';
+import TradingPlatformIcon from 'Assets/wallets';
+import React from 'react';
+
+type TStep = {
+    icon: JSX.Element;
+    title: string;
+    description: string;
+    bullets: string[];
+};
+
+export const steps = (eu_user: boolean): TStep[] => [
+    {
+        icon: <TradingPlatformIcon icon={eu_user ? 'IntroducingWalletsEU' : 'IntroducingWallets'} />,
+        title: localize('Introducing Wallets'),
+        description: localize('A better way to manage your funds'),
+        bullets: [
+            localize('One Wallet, one currency'),
+            localize('A Wallet for each currency to focus your funds'),
+            !eu_user && localize('Get one Wallet, get several - your choice'),
+        ],
+    },
+    {
+        icon: <TradingPlatformIcon icon='HowItWorks' />,
+        title: localize('How it works'),
+        description: localize('Get a Wallet, add funds, trade'),
+        bullets: [
+            localize('Get a Wallet for the currency you want'),
+            localize('Add funds to your Wallet via your favourite payment method'),
+            localize('Move funds to your trading account to start trading'),
+        ],
+    },
+    {
+        icon: <TradingPlatformIcon icon='TradingAccounts' />,
+        title: localize('What happens to my trading accounts'),
+        description: localize("We'll link them"),
+        bullets: [
```

Comment:
suppose it's better to define a body for each step since for the rest we do not have this bullets part and the contnet is different

Reply:
@mahdiyeh-deriv I dont think the other steps can use my component. Because its entirely different 🤔